packages/nx/src/config/project-graph.ts

```diff
@@ -25,8 +25,8 @@ export interface ProjectFileMap {
 /**
  * A Graph of projects in the workspace and dependencies between them
  */
-export interface ProjectGraph<T = any> {
-  nodes: Record<string, ProjectGraphProjectNode<T>>;
+export interface ProjectGraph {
```

Comment:
Removing the generic type here is fine, but breaking for community plugin's CI / build. We should migrate it, since plugins runtime code isn't affected its not a huge deal but something we should still likely address.